docs/src/getting-started/verification-results/src/main.rs

```diff
@@ -2,6 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 
 #[kani::proof]
+#[kani::unwind(4)]
```

Comment:
Why we didn't need to unwind these cases before?

Reply:
I still haven't had time to investigate why

Reply:
I created #3088 for us to investigate what can be done to remove these attributes.